server/grpc_service.go

```diff
@@ -569,6 +590,72 @@ func (s *GrpcServer) Tso(stream pdpb.PD_TsoServer) error {
 			continue
 		}
 
+		if s.forwardToTSOService() {
+			if request.GetCount() == 0 {
```

Comment:
Can we reuse `forwardTSO` here?

Reply:
Let me think about how to reduce the duplicated code.